CP-CPS.md

```diff
@@ -817,7 +817,7 @@ If a suitable successor entity does not exist, the following steps will be taken
 
 ### 6.1.1 Key pair generation
 
-ISRG CA Private Keys are generated by HSMs meeting the requirements of Section 6.2.1. This occurs during a ceremony meeting the requirements of this CP/CPS.
+ISRG CA Private Keys are generated by HSMs as described in [Section 6.2.1](#621-cryptographic-module-standards-and-controls). This occurs during a ceremony meeting the requirements of this CP/CPS.
```

Comment:
"as described in" makes it sound like Section 6.2.1 is going to describe the process by which the CA private keys are generated, when in reality it's just HSM specifications. I think the original text makes more sense here, but I see what you were trying to do here and there's probably just a better way to say it, like:

"ISRG CA Private Keys are generated by HSMs meeting the specifications described in [Section 6.2.1](#621-cryptographic-module-standards-and-controls)."